### PR TITLE
Add Feed Test

### PR DESCRIPTION
**Pull request by araskachoi**
## Changes

- 
- 
-

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -84,7 +85,7 @@ import (
 	dbm "github.com/tendermint/tm-db"
 )
 
-const Name = "ChainLink"
+const Name = "chainlink"
```
Comment (freddyli7) [inline app/app.go:88]: does the app name need to be app lower case?

File: scripts/addFeed.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+aliceAddr=$(chainlinkd keys show alice -a)
+alicePK=$(chainlinkd keys show alice -p)
+
+bobAddr=$(chainlinkd keys show bob -a)
+bobPK=$(chainlinkd keys show bob -p)
+
+cerloAddr=$(chainlinkd keys show cerlo -a)
+cerloPK=$(chainlinkd keys show cerlo -p)
+
+# aDd NeW fEeD bY aLiCe
```
Comment (freddyli7) [inline scripts/addFeed.sh:16]: what's going on here with the case?

Comment (araskachoi) [reply scripts/addFeed.sh:16]: woops, i was just in a frustrated mood LOL

File: app/app.go (hunk 1 above)
Comment (araskachoi) [reply app/app.go:88]: yes, it should be all lower case, if there is an err with the cli cmd, it'll give the wrong cmd name

File: x/chainlink/client/cli/txFeed.go [hunk 3]
```diff
@@ -17,7 +17,8 @@ import (
 func CmdAddFeed() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "addFeed [feedId] [feedOwnerAddress] [submissionCount] [heartbeatTrigger] [deviationThresholdTrigger] [initDataProviderList]",
-		Short: "Add new feed. Signer must be the existing module owner. initDataProviderList is a string contains each data provider's address with pubkey and split by comma",
+		Short: "Add new feed. Signer must be the existing module owner.",
+		Long:  "The following fields are required:\n\tThe feedId will be a string that uniquely identifies the feed. The feedOwnerAddress must be a valid cosmos address.\n\tThe submissionCount in the required number of signatures.\n\tThe deviationThresholdTrigger is the fraction of deviation in the feed data required to trigger a new round.\n\tThe initDataProviderList is a string contains each data provider's address with pubkey and split by comma.",
```
Comment (freddyli7) [inline x/chainlink/client/cli/txFeed.go:21]: 👏

File: app/app.go (hunk 1 above)
Comment (freddyli7) [reply app/app.go:88]: good to know, tHanKs @araskachoi 😆

**Review by freddyli7 — APPROVED** [review]
LGTM!

**Pull request merged**